.travis.yml

```diff
@@ -0,0 +1,63 @@
+# After changing this file, check it on http://yaml-online-parser.appspot.com/
+
+language: python
+
+env:
+  global:
+    - MINICONDA_VERSION="latest"
+    - MINICONDA_LINUX="Linux-x86_64"
+    - MINICONDA_OSX="MacOSX-x86_64"
+
+matrix:
+  include:
+    # Linux using miniconda
+    - python: "2.7"
+      env: USE_QT_API=PyQt4 USE_CONDA=true
+      os: linux
+    - python: "3.3"
+      env: USE_QT_API=PyQt4 USE_CONDA=true
+      os: linux
+    - python: "3.4"
+      env: USE_QT_API=PyQt4 USE_CONDA=true
+      os: linux
+    #- python: "2.7"
+    #  env: USE_QT_API=PyQt5 USE_CONDA=true
+    #  os: linux
+    #- python: "3.3"
+    #  env: USE_QT_API=PyQt5 USE_CONDA=true
+    #  os: linux
+    #- python: "3.4"
+    #  env: USE_QT_API=PyQt5 USE_CONDA=true
+    #  os: linux
+    #- python: "2.7"
+    #  env: USE_QT_API=PySide USE_CONDA=true
+    #  os: linux
+    # Linux using apt-get/pip packages
+    - python: "2.7"
+      env: USE_QT_API=PyQt4 USE_CONDA=false
+      os: linux
+    - python: "2.7"
+      env: USE_QT_API=PySide USE_CONDA=false
+      os: linux
+    - python: "3.3"
+      env: USE_QT_API=PySide USE_CONDA=false
+      os: linux
+    - python: "3.4"
+      env: USE_QT_API=PySide USE_CONDA=false
+      os: linux
+
+before_install:
+  - sudo apt-get update
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+
+install:
+  - ./continuous_integration/travis_install.sh;
+  - export PATH="$HOME/miniconda/bin:$PATH";
```

Comment:
Not exactly, but I see its purpose now. This line is needed to add `miniconda/bin` to PATH so that `source activate` works.